lib/models/project.js

```diff
@@ -311,7 +311,7 @@ class Project {
     if (fs.existsSync(`${targetsPath}.js`)) {
       this._targets = this.require(targetsPath);
     } else {
-      this._targets = require('../utilities/default-targets');
+      this._targets = require('../../blueprints/app/files/config/targets');
```

Comment:
Or is this trying to be too clever?
I know some blueprint files aren't valid JS because of the string interpolation, but here it works.